third_party/blink/renderer/modules/global_privacy_control/navigator_global_privacy_control.cc

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/third_party/blink/renderer/modules/global_privacy_control/navigator_global_privacy_control.h"
+
+#include "third_party/blink/renderer/core/frame/navigator.h"
+
+namespace blink {
+
+NavigatorGlobalPrivacyControl::
+  NavigatorGlobalPrivacyControl(Navigator& navigator) // NOLINT
+    : Supplement<Navigator>(navigator) {}
+
+// static
+const char NavigatorGlobalPrivacyControl::kSupplementName[] =
+    "NavigatorGlobalPrivacyControl";
+
+NavigatorGlobalPrivacyControl&
+  NavigatorGlobalPrivacyControl::From(Navigator& navigator) {
+  NavigatorGlobalPrivacyControl* supplement =
+      Supplement<Navigator>::From<NavigatorGlobalPrivacyControl>(navigator);
+  if (!supplement) {
+    supplement = MakeGarbageCollected<NavigatorGlobalPrivacyControl>(navigator);
+    ProvideTo(navigator, supplement);
+  }
+  return *supplement;
+}
+
+bool NavigatorGlobalPrivacyControl::
+  globalPrivacyControl(blink::Navigator& navigator) { // NOLINT
+  return true;
```

Comment:
I believe we want to check `base::FeatureList::IsEnabled` here

Reply:
@bbondy I discussed this with @iefremov and @bridiver in DMs - there isn't a way to gate the _presence_ of the IDL-generated API by a feature flag. We've decided it's fine since this is purely a new API and shouldn't break webcompat anywhere. And if it has to exist, better to just always return `true` rather than `false`.

Reply:
ok